app/controllers/clinical_resource_hubs_controller.rb

```diff
@@ -20,31 +20,6 @@ def show
     get_categories_by_practices(@practices_adopted_by_crh, @crh_practices_adopted_categories)
   end
 
-  # GET /crh/:id/created_crh_practices
-  def created_crh_practices
```

Comment:
Let's remove this from the `before_action` in line 4 as well

Reply:
good eye, thank you for catching that!